src/ssl.c

```diff
@@ -5412,6 +5412,7 @@ int ProcessBuffer(WOLFSSL_CTX* ctx, const unsigned char* buff,
 {
     DerBuffer*    der = NULL;        /* holds DER or RAW (for NTRU) */
     int           ret = 0;
+    int           done = 0;
```

Comment:
This function appears based on white-space changes to have had a massive overhaul. When viewing without the whitespace changes it's pretty minor (just adding 5907-5917). This is an important and complex function. Perhaps you can update without so much white space change?